setup.py

```diff
@@ -0,0 +1,11 @@
+from setuptools import find_packages, setup
+
+setup(
+    name="macrophase-striation-analysis",
+    version="0.0.1",
+    author="Erik Serrano, Way Lab",
```

Comment:
```suggestion
    author="Erik Serrano",
```

I don't think you need this. It might belong more suitably in an `affiliation` flag maybe?